test/files/rules/sortedkey.test.ts

```diff
@@ -0,0 +1,61 @@
+var passA = {
+    a: 1,
+    b: 2
+};
+
+var failA = {
+    b: 1,
+    a: 2
+};
+
+var passB = {
+    a: 1,
+    b: 2,
+    c: 3,
+    d: 4
+};
+
+var failB = {
+    c: 3,
+    a: 1,
+    b: 2,
+    d: 4
+};
+
+var passC = {
+    a: 1,
+    b: {
+        aa: 1,
+        bb: 2
+    }
+};
+
+var failC = {
+    a: 1,
+    b: {
+        bb: 2,
```

Comment:
Can you add a nesting test that's like, say, {a: 1, c: {aa: 1, bb: 2}, b: 3} or somesuch -- that is, a test that checks the thing from before we went into the next layer down?